src/github/github.js

```diff
@@ -230,22 +235,27 @@ const textMatchesJankIndex = textMatchesString('jank:');
 const textMatchesShippit = textMatchesAnyString(shippedEmojis);
 const textMatchesUpdate = textMatchesAnyString(updateMeEmojis);
 
-const prsToTriggered = async (textFilter, pullReqs) => {
-  const prs = (await getOpenPrs(pullReqs)).filter(pr => {
+const prsToTriggered = async (textFilter, pullReqs, filterByLabelName) => {
```

Comment:
prefer to see `filterByLabelName = false` and then only passing it in from callers when it's required.